src/js/disability-benefits/526EZ/pages/primaryAddress.js

```diff
@@ -0,0 +1,642 @@
+  { label: 'Nebraska', value: 'NE' },
+  { label: 'Nevada', value: 'NV' },
+  { label: 'New Hampshire', value: 'NH' },
+  { label: 'New Jersey', value: 'NJ' },
+  { label: 'New Mexico', value: 'NM' },
+  { label: 'New York', value: 'NY' },
+  { label: 'North Carolina', value: 'NC' },
+  { label: 'North Dakota', value: 'ND' },
+  { label: 'Northern Mariana Islands', value: 'MP' },
+  { label: 'Ohio', value: 'OH' },
+  { label: 'Oklahoma', value: 'OK' },
+  { label: 'Oregon', value: 'OR' },
+  { label: 'Palau', value: 'PW' },
+  { label: 'Pennsylvania', value: 'PA' },
+  { label: 'Puerto Rico', value: 'PR' },
+  { label: 'Rhode Island', value: 'RI' },
+  { label: 'South Carolina', value: 'SC' },
+  { label: 'South Dakota', value: 'SD' },
+  { label: 'Tennessee', value: 'TN' },
+  { label: 'Texas', value: 'TX' },
+  { label: 'Utah', value: 'UT' },
+  { label: 'Vermont', value: 'VT' },
+  { label: 'Virgin Islands', value: 'VI' },
+  { label: 'Virginia', value: 'VA' },
+  { label: 'Washington', value: 'WA' },
+  { label: 'West Virginia', value: 'WV' },
+  { label: 'Wisconsin', value: 'WI' },
+  { label: 'Wyoming', value: 'WY' }
+];
+
+const typeLabels = {
+  DOMESTIC: 'Domestic',
+  MILITARY: 'Military',
+  INTERNATIONAL: 'International'
+};
+
+export const militaryPostOfficeTypeLabels = {
+  APO: 'Army Post Office',
+  FPO: 'Fleet Post Office',
+  DPO: 'Diplomatic Post Office'
+};
+
+const requiredFields = ['country', 'addressLine1'];
+
+function isValidZIP(value) {
+  if (value !== null) {
+    return /^\d{9}$/.test(value) || /^\d{5}-\d{4}$/.test(value);
+  }
+  return true;
+}
+
+function isValidPhone(value) {
+  if (value !== null) {
+    return /^\d{10}$/.test(value) || /^\d{11}$/.test(value);
+  }
+  return true;
+}
+
+function validatePhone(errors, phone) {
+  if (phone && !isValidPhone(phone)) {
+    errors.addError(
+      'Phone numbers must be at least 10 digits (dashes allowed)'
+    );
+  }
+}
+
+function validateZIP(errors, zip) {
+  if (zip && !isValidZIP(zip)) {
+    errors.addError('Please enter a valid 9 digit ZIP (dashes allowed)');
+  }
+}
+
+const addressSchema = {
+  type: 'object',
+  required: requiredFields,
+  properties: {
+    type: {
+      'default': 'DOMESTIC',
+      type: 'string',
+      'enum': ['DOMESTIC', 'MILITARY', 'INTERNATIONAL']
+    },
+    country: {
+      type: 'string'
+    },
+    addressLine1: {
+      type: 'string',
+      maxLength: 35,
+      pattern: "([a-zA-Z0-9-'.,,&#]([a-zA-Z0-9-'.,,&# ])?)+$"
+    },
+    addressLine2: {
+      type: 'string',
+      maxLength: 35,
+      pattern: "([a-zA-Z0-9-'.,,&#]([a-zA-Z0-9-'.,,&# ])?)+$"
+    },
+    addressLine3: {
+      type: 'string',
+      maxLength: 35,
+      pattern: "([a-zA-Z0-9-'.,,&#]([a-zA-Z0-9-'.,,&# ])?)+$"
+    },
+    city: {
+      type: 'string',
+      maxLength: 35,
+      pattern: "([a-zA-Z0-9-'.#]([a-zA-Z0-9-'.# ])?)+$"
+    },
+    state: {
+      type: 'string',
+      'enum': states
+    },
+    zipCode: {
```

Comment:
This is presented as a single field as that is our standard pattern, would it make sense to convert this on the BE or in transform for submit, or should a widget be created to split this up into zipFirstFive and zipLastFour?